### PR TITLE
[Hint Mode: Start Coords] Add start coords UI for sinusoid graphs

### PR DESCRIPTION
**Pull request by nishasy**
## Summary:
Add the UI to specify start coords for Sinusoid graph type.

- Add the sinusoid graph type to start-coord-settings.tsx
- Create a start-coords-sinusoid.tsx file with the main UI
- Add a utility for getting the sinusoid equation from coordinates (less complex
  than the static method on InteractiveGraph that does the same)
- Add start coords UI phase 2 flag

Issue: https://khanacademy.atlassian.net/browse/LEMS-2207

## Test plan:
`yarn jest`

Storybook
- http://localhost:6006/?path=/story/perseuseditor-widgets-interactive-graph--interactive-graph-sinusoid-with-starting-coords

### CODE REVIEW

File: packages/perseus-editor/src/__stories__/flags-for-api-options.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const flags = {
         // Start coords UI flags
         // TODO(LEMS-2228): Remove flags once this is fully released
         "start-coords-ui-phase-1": true,
+        "start-coords-ui-phase-2": true,
```
Comment (nishasy) [inline packages/perseus-editor/src/__stories__/flags-for-api-options.ts:22]: I can remove this and all the extra flag code below if we want to release sinusoid as part of phase 1.

**Review by mark-fitzgerald — APPROVED** [review]
Looks good.

File: packages/perseus-editor/src/components/start-coords-sinusoid.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                    coord={startCoords[1]}
+                    labels={["x", "y"]}
+                    onChange={(value) => onChange([startCoords[0], value])}
+                />
+            </View>
+        </>
+    );
+};
+
+const styles = StyleSheet.create({
```
Comment (mark-fitzgerald) [inline packages/perseus-editor/src/components/start-coords-sinusoid.tsx:58]: NIT: Grrr. Same styling as all the [other "start-coords-*" implementations](https://github.com/Khan/perseus/pull/1469/files#diff-818cce2b94b793838c115d033d11f3235fd9f9e2696b18060a95b1ac2a89a435R73-R93).
No need to change anything in this PR.

**Review by mark-fitzgerald — COMMENTED** [review]
LGTM

**Pull request merged**